spec/acceptance/integration_ntp_spec.rb

```diff
@@ -27,26 +27,26 @@ class { 'ntp': }
     after(:all) { reset_target_host }
 
     describe 'go to the future' do
-      it 'install ntpdate and check its 2023' do
+      it 'install ntpdate and check the year is current' do
         apply_manifest("package { 'ntpdate': ensure => present }")
         result = run_shell('date')
-        expect(result.stdout).to match(%r{2023})
+        expect(result.stdout).to match(%r{#{Time.now.year}})
       end
 
       it 'disable ntp auto-sync' do
         result = run_shell('timedatectl set-ntp false')
         expect(result.exit_code).to eq(0)
       end
 
-      it 'go forward to 2024' do
+      it 'go forward to to next year' do
         result = run_shell('date --set="$(date --date="next year")"')
-        expect(result.stdout).to match(%r{2024})
+        expect(result.stdout).to match(%r{#{Time.now.year+1}})
```

Comment:
I think you need to include a couple of spaces here to keep Rubocop happy
```suggestion
        expect(result.stdout).to match(%r{#{Time.now.year + 1}})
```